src/test/java/org/broadinstitute/hellbender/engine/filters/ReadFilterPluginUnitTest.java

```diff
@@ -164,6 +164,15 @@ public void testEnableNonExistentFilter() {
         clp.parseArguments(System.out, new String[] {"--RF", "fakeFilter"});
     }
 
+    @Test(expectedExceptions = CommandLineException.BadArgumentValue.class)
+    public void testEnableTwiceFilter() {
+        CommandLineParser clp = new CommandLineArgumentParser(new Object(),
+                Collections.singletonList(new GATKReadFilterPluginDescriptor(null)));
+        clp.parseArguments(System.out, new String[] {
+                "--RF", ReadLengthReadFilter.class.getSimpleName(),
+                "--RF", ReadLengthReadFilter.class.getSimpleName()});
```

Comment:
Can you add another test case that includes both a duplicate and a non duplicate (through a data provider). thx.